package.json

```diff
@@ -18,16 +19,24 @@
     "prepack": "nuxt-module-build",
     "dev": "nuxi dev playground",
     "dev:build": "nuxi build playground",
-    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground"
+    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
+    "release": "npm run test && changelogen --release && git push --follow-tags && npm publish",
```

Comment:
```suggestion
    "release": "npm run test && changelogen --release && npm publish && git push --follow-tags",
```

Reply:
I've seen it put before like this on other repos, but I'm curious what's the reason for it?

Reply:
It is almost the same. The only different is that, over over time it was proven to publish first is safer. so if publish failed we don't push a tag that needs manually removing from remote).

Reply:
OK, got it!

Reply:
Done in [41f8d12](https://github.com/nuxt/starter/pull/216/commits/41f8d12609a314fa1dfb15a4b295ee0c48539053)